signal/signal.go

```diff
@@ -85,10 +91,12 @@ func executeRunner(c config.Config) error {
 		select {
 		case r := <-done:
 			if len(c.TestURL) > 0 {
-				log.Info("Adding test data")
+				log.Debugf("Adding test data for %s: %+v", r.getName(), r.getTrack())
 				tester[r.getName()] = r.getTrack()
 			} else if len(r.getError()) > 0 {
-				log.Errorf("%s: %s", r.getName(), r.getError())
+				for _, err := range r.getError() {
+					log.Errorf("%s: %s", r.getName(), err)
```

Comment:
i am not quite sure what benefits it gives you to collect the errors and then log them all.
Maybe it is better to log the error right away? Might be more useful when debugging